.travis.yml

```diff
@@ -0,0 +1,30 @@
+---
+dist: trusty
+language: python
+python: "2.7"
+
+# Use isolated GCE instance
+sudo: required
+
+env:
+  global:
+    - ANSIBLE_CONFIG=tests/ansible.cfg
+
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install python-pip python-dev ca-certificates -qq
+
+install:
+  - pip install ansible==2.3.0.0
+  - pip install urllib3
+  - ansible --version
+  # Install additional packages, configure lxc, start streisand container
+  - ansible-playbook tests/development-setup.yml -i tests/inventory
+  # Ensure we get added to the lxd group
+  - sudo chmod 777 /var/lib/lxd/unix.socket
```

Comment:
The comment makes a mention of an `lxd` group, but this `chmod` seems to be setting world read/write/execute for everyone to this socket. Should the comment be updated or the `chmod` replaced with something group related?